app/src/main/res/xml/preferences.xml

```diff
@@ -11,10 +11,11 @@
             android:icon="@drawable/ic_network_wifi_grey600_24dp"
             android:key="@string/pref_key_wifi_only"
             android:layout="@layout/item_pref"
-            android:title="@string/settings_wifi_only" />
+            android:title="@string/settings_wifi_only"
+            app:defaultValue="true" />
 
         <ListPreference
-            android:defaultValue="@string/pref_value_sync_off"
+            android:defaultValue="@string/pref_value_sync_15_minutes"
```

Comment:
default value should not be one of thr strings in the future, but simply `true` - a boolean. Since we used a string in older app versions, we need the migration 🙂